app/controllers/app.py

```diff
@@ -934,6 +940,378 @@ def delete(self, app_id):
         except Exception as e:
             return dict(status='fail', message=str(e)), 500
 
+    
+    @admin_required
```

Comment:
This does not need to been an admin only resource.

Reply:
Seems like Allan was trying to separate the admin and user endpoints